ADALiOS/ADALiOS/ADAuthenticationRequest+WebRequest.m

```diff
@@ -234,6 +237,29 @@ - (void)requestWithServer:(NSString *)authorizationServer
                     }
                 }
                     break;
+                case 500:
+                case 503:
```

Comment:
Now the retry behaviour all happens at the web request level. Authentication request does not know about this.